src/pytorch_lightning/strategies/hpu_parallel.py

```diff
@@ -151,3 +151,13 @@ def teardown(self) -> None:
         # Was set to local rank
         os.environ.pop("ID", None)
         os.environ.pop("HCCL_DISTRIBUTED_BACKEND", None)
+
+    @property
```

Comment:
Please add the properties and setters to the top, after init :) 
Req. change for this so we don't diverge further from the current organization